docs/v1.X/docs/references/plugins.md

```diff
@@ -0,0 +1,92 @@
+# Understanding Plugins
```

Comment:
only `# Plugins` or `# Plugins Format` to be complient with other references

Reply:
Thanks !. fixed via https://github.com/crowdsecurity/crowdsec-docs/pull/27/commits/e2c793c28d53f5523b7d184fb9a1ca077da319d2